IntegrationTests.Dockerfile

```diff
@@ -13,6 +13,6 @@ COPY tests/requirements.txt /opt
 RUN pip install --no-cache-dir -r /opt/requirements.txt
 
 WORKDIR /app
-COPY tests .
+COPY tests/ pyproject.toml ./
 
-ENTRYPOINT ["/bin/bash", "/app/run.sh"]
+ENTRYPOINT ["/app/run.sh"]
```

Comment:
Could you add a comment that this is `tests/run.sh` in the repo? (and not `bin/run.sh`)

Reply:
[3f703a9](https://github.com/mozilla/remote-settings/pull/378/commits/3f703a9d11e237205cc6d4a5c31426289e70ae49)